DashKit.xcodeproj/project.pbxproj

```diff
@@ -1611,7 +1611,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 75U4X84TEG;
```

Comment:
[nit] intentional change?

Reply:
Yes, this is the Tidepool ID; so it should work for all of us, when compiling the PodUIDemo app.